run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
```

Comment:
Shouldn't this be run_systemd_container?  I would think users are going to want to be able to just run regular containers with kpod also.